nextcord/client.py

```diff
@@ -1467,6 +1468,31 @@ async def fetch_guild(self, guild_id: int, /, *, with_counts: bool = True) -> Gu
         data = await self.http.get_guild(guild_id, with_counts=with_counts)
         return Guild(data=data, state=self._connection)
 
+    async def fetch_guild_preview(self, guild_id: int, /) -> GuildPreview:
+        """|coro|
+
+        Fetches a :class:`.GuildPreview` from an ID.
```

Comment:
You should mention that only guilds that the bot is in or lurkable guilds can be fetched.